docs/first-steps.md

```diff
@@ -33,7 +33,7 @@ These commands are used to manage Python itself. uv is capable of installing and
 - `uv python list`
 - `uv python find`
 
-See the documentation on [toolchains](./preview/toolchains.md) for more details on getting started.
+See the documentation on [toolchains](./python-versions.md) for more details on getting started.
```

Comment:
I think?

Reply:
Maybe?

```suggestion
See the documentation on [Python version management](./python-versions.md) for more details on getting started.
```

Or maybe the Python guide? I can own fixing this eventually — lots to do in this doc.